### PR TITLE
Revert "Revert "david lewine added his page and updated index""

### PR DESCRIPTION
**Pull request by davidlewine**
This reverts commit 80642be45373e0a7847f28e0fb89e3f036fa857b.

### CODE REVIEW

**Comment by SamShapiro** [comment]
lgtm

**Comment by bfauble** [comment]
lgtm

**Comment by davidlewine** [comment]
wow, nice work!

**Comment by nw0428** [comment]
Looks good. Why did you copy my page?

File: summer_2015/davidlpage.html [hunk 1]
```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- This line is required for media queries (IE, makes your shit responsive) -->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Meta info about site -->
+    <title>David LeWine, Spice Wizard of the Cosmic Snowflake</title>
```
Comment (nw0428) [inline summer_2015/davidlpage.html:7]: You are also a spice Wizard?

**Comment by carlos4z** [comment]
Well done

**Comment by jcb982** [comment]
lgtm

File: summer_2015/davidlpage.html [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    <meta name="description" content="the hero Iowa wants, but Boston needs: GRILLMAN">
+    <meta name="author" content="Davood">
+
+    <!-- CSS Files. Try and have as few of these as possible -->
+    <link rel="stylesheet" href="../spring_2015/css/david/bootstrap.css" type="text/css">
```
Comment (maxamilion) [inline summer_2015/davidlpage.html:12]: wow

**Comment by maxamilion** [comment]
lgtm

**Pull request merged**